.eslintrc

```diff
@@ -4,20 +4,30 @@
     "strict": 0,
     "no-underscore-dangle": 0,
     "no-unused-vars": 0,
-    "curly": 0,
     "no-multi-spaces": 0,
     "key-spacing": 0,
     "no-return-assign": 0,
-    "consistent-return": 0,
+    "consistent-return": 1,
     "no-shadow": 0,
-    "no-comma-dangle": 0,
-    "no-use-before-define": 0,
-    "no-empty": 0,
-    "new-parens": 0,
-    "no-cond-assign": 0,
-    "quotes": "single"
+    "comma-dangle": [1, "never"],
```

Comment:
@Globegitter How do you feel about `"comma-dangle": [2, "always"],`? 

This would ensure that all object literals and array literals end in a dangling comma (to prevent errors when adding items).  It is valid ECMAScript, and the only reason not to use them seems to be IE8, which we don't care about (http://eslint.org/docs/rules/comma-dangle.html).   The code is not written this way currently, so it would take a little work to make the test pass, but that's something I could tackle.  

```
var foo = {
    bar: "baz",
    qux: "quux",
};
```